src/selectedtests/task_mappings/create_task_mappings.py

```diff
@@ -77,6 +78,12 @@ def create_task_mappings(
             jobs = []
             with Executor(max_workers=MAX_WORKERS) as exe:
                 for next_version, version, prev_version in windowed_iter(project_versions, 3):
+                    if not most_recent_version_analyzed:
+                        most_recent_version_analyzed = version.version_id
+
+                    if version.version_id == after_version:
```

Comment:
The name `after_version` doesn't seem very clear to me. Maybe something like "stop version" or "stopping version" would work better?

Reply:
Is it clearer now that I've refactored, or still unclear?